integrationTests/chainSimulator/bridge/common.go

```diff
@@ -98,7 +98,7 @@ func deployBridgeSetup(
 	esdtSafeAddress := deployEsdtSafeContract(t, cs, ownerAddrBytes, &nonce, systemContractDeploy, argsEsdtSafe, contractWasmPath)
 
 	feeMarketArgs := "@" + hex.EncodeToString(esdtSafeAddress) + // esdt_safe_address
-		"@00" // no fee
+		"@010000000d4e5445524e532d656161643135010000000d4e5445524e532d6561616431350000000405f5e1000000000405f5e100" // no fee
```

Comment:
Is the comment "no fee" still relevant?

Reply:
I didn't mean to push this, it was just a debugging session.
It should be `@00` // no fee